bin/node-pg-migrate

```diff
@@ -240,21 +246,20 @@ let tsconfigPath = argv[tsconfigArg]
 function readTsconfig() {
   if (tsconfigPath) {
     let tsconfig
-    let tsnode
+    const json5 = tryRequire('json5')
+
     try {
       // eslint-disable-next-line security/detect-non-literal-fs-filename
       const config = fs.readFileSync(path.resolve(process.cwd(), tsconfigPath), { encoding: 'utf-8' })
-      // eslint-disable-next-line global-require,import/no-dynamic-require,security/detect-non-literal-require,import/no-extraneous-dependencies
-      tsconfig = require('json5').parse(config)
+      tsconfig = json5 ? json5.parse(config) : JSON.parse(config)
     } catch (err) {
       console.error("Can't load tsconfig.json:", err)
```

Comment:
**suggestion (if-minor, dx):** We should improve this message, because it is misleading and confuse the developer  
Actually it is more like "Reading the `tsconfig.json` file failed due to JSON parse error" or something similar

Reply:
Well, the `Can't load tsconfig.json:` currently stands for both reading and parsing errors. Both the `readFileSync` and `JSON.parse` (resp. `json5.parse`) are within this `try/catch`.

Since we're logging the actually cause for the error as well, I would assume that the two cases are easily distinguishable :shrug: 

I'm open to more arguments though.